src/v/coproc/event_listener.cc

```diff
@@ -13,25 +13,23 @@
 #include "config/configuration.h"
 #include "coproc/errc.h"
 #include "coproc/logger.h"
-#include "coproc/types.h"
 #include "coproc/wasm_event.h"
 #include "ssx/future-util.h"
 #include "storage/parser_utils.h"
-#include "utils/file_io.h"
 #include "utils/unresolved_address.h"
 #include "vassert.h"
 #include "vlog.h"
 
+#include <seastar/core/coroutine.hh>
 #include <seastar/core/future.hh>
-#include <seastar/core/gate.hh>
 #include <seastar/core/loop.hh>
 #include <seastar/core/seastar.hh>
-#include <seastar/core/sstring.hh>
-#include <seastar/net/socket_defs.hh>
 
 namespace coproc::wasm {
 
-using namespace std::chrono_literals;
+/// This may never change as it is not a configurable value
+static const model::topic_partition coproc_internal_tp{model::topic_partition(
+  model::topic("coprocessor_internal_topic"), model::partition_id(0))};
```

Comment:
this should probably go into the namespace.h file with other well-known topic names.